test_utils/__init__.py

```diff
@@ -1,10 +1,56 @@
 """
 Test utilities.
+"""
 
-Since pytest discourages putting __init__.py into testdirectory
-(i.e. making tests a package) one cannot import from anywhere
-under tests folder. However, some utility classes/methods might be useful
-in multiple test modules (i.e. factoryboy factories, base test classes).
+from rest_framework.test import APIClient, APITestCase
 
-So this package is the place to put them.
-"""
+from test_utils.factories import UserFactory
+from test_utils.mixins import JwtMixin
+
+TEST_USERNAME = 'api_worker'
+TEST_EMAIL = 'test@email.com'
+TEST_PASSWORD = 'QWERTY'
+TEST_LMS_USER_ID = 5
```

Comment:
Is this an okay way to set the lms user id?

Reply:
since it's just a number now it's ok to just put in a number. I'd probably control it inside the individual test though since many tests would be interested in running across different users.

Reply:
Makes sense on setting that as a number for now. So my thought on having the test variable was (I had been referencing the tests in edx-exams) that the test_utils would be used to generate more generic user cases, then users could also be created in the individual test cases (I did this for invalid program intents). In this case, if I wanted to add more users, I could just add another user in test_utils or I could just create in the individual test case? Does that make sense? Or is there another reason to only set the lms_user_id in the individual test case?

Reply:
I think this is a personal taste thing, it's not super important either way. These days I tend to put more in the test case and less in other places just because I don't like going through a bunch of files trying to figure out where the 5 in my output came from.

Reply:
I see that makes sense! For now, I'll leave as is, since there's only one user and to keep it consistent with Marcin's endpoint ticket as well. But that's something I'll think about for the future. Definitely helpful to understand the pros/cons.